src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -610,13 +611,38 @@ protected function selectTableNames(string $databaseName): Result
 
     protected function selectTableColumns(string $databaseName, ?string $tableName = null): Result
     {
+        $fragmentDefaultValueSQL = <<<'SQL'
+             SELECT pg_get_expr(adbin, adrelid)
+             FROM pg_attrdef
+             WHERE c.oid = pg_attrdef.adrelid
+                AND pg_attrdef.adnum=a.attnum
+        SQL;
+
+        if ($this->_platform instanceof PostgreSQL120Platform) {
```

Comment:
Everything inside this `if` should be moved inside `PostgreSQL120Platform`. We should not need to perform any `instanceof` checks here.

Reply:
fair enough